main.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "report_builder"
+
+options = {
+  report_title: 'Smoke Testing Report',
+  report_types: ['json', 'html']
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mpermperpisang/golang-automation/blob/4c37372fb449e752801dcb55d55b269bb510a466/main.rb#L7"></span>Use <code>%w</code> or <code>%W</code> for an array of words.</td>
    </tr>
  </tbody>
</table>